code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -454,6 +454,19 @@
 	tail = "armalis_tail"
 	icon_template = 'icons/mob/human_races/r_armalis.dmi'
 
+	atmos_requirements = list(
```

Comment:
Please do not change the armalis, they are already buggy, inconsistent, can't wear most masks, and barely function.